### PR TITLE
Fix the issue with token revocation when account is locked/disabled

### PR DESCRIPTION
**Pull request by piraveena**
### Proposed changes in this pull request

Frontport https://github.com/wso2-support/identity-event-handler-account-lock/pull/35

### CODE REVIEW

**Comment by piraveena** [comment]
PR builder passed successfully https://github.com/wso2/product-is/actions/runs/357237004

File: components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/constants/AccountConstants.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
     public static final String LOCKED = "LOCKED";
     public static final String UNLOCKED = "UNLOCKED";
     public static final String DISABLED = "DISABLED";
-
+    public static final String USER_ACCOUNT_STATE_WITH_USERNAME = "UserAccountStateWithUsername";
```
Comment (sachiniWettasinghe) [inline components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/constants/AccountConstants.java:54]: Any reason to add this constant? Can't we use https://github.com/wso2/carbon-identity-framework/pull/3216/files#diff-5398caecd73528910fe590141c693a88b807d6f029c2f16e61866e62515b70d3R66?

**Review by IsuraD — APPROVED** [review]

File: components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountDisableHandler.java [hunk 2]
```diff
@@ -234,6 +234,9 @@ protected boolean handlePreSetUserClaimValues(Event event, String userName, User
                     IdentityUtil.setIdentityErrorMsg(customErrorMessageContext);
                     IdentityUtil.threadLocalProperties.get().put(IdentityCoreConstants.USER_ACCOUNT_STATE,
                             IdentityCoreConstants.USER_ACCOUNT_DISABLED_ERROR_CODE);
+                    IdentityUtil.threadLocalProperties.get()
```
Comment (IsuraD) [inline components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountDisableHandler.java:237]: where do we clear these thread locals if the OAuth event listener is disabled?

**Comment by piraveena** [comment]
closing this PR since adding a new threadlocal approach is not preferebale.Due to this created a new eventHandler in oauth https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1501

**Pull request closed without merging**